.github/workflows/tests.yml

```diff
@@ -299,7 +299,7 @@ jobs:
     - name: Use unix line endings
       shell: bash
       run: |
-        # Github's actions/checkout@v2 when run on Windows mangles the line-endings to DOS-style
+        # Github's actions/checkout@v3 when run on Windows mangles the line-endings to DOS-style
```

Comment:
The linked issue seems to still be revelant with `v3`, so updating this comment seems fine :+1: